delete unused sql functions

snopoke opened the pull request:
Waiting on https://github.com/dimagi/commcare-hq/pull/17148 to be deployed

Commit:
Hide detail tile continue correctly

snopoke committed:
remove save_ledger_values function

snopoke committed:
remove hard_delete_forms SQL

snopoke committed:
use correct exception
https://sentry.io/dimagi/commcarehq/issues/253769300/

benrudolph reviewed (approved):
nice 👍 when the other PR is merged

emord committed:
Removed unused imports.

dannyroberts committed:
fix report builder for unicode (french support)

emord committed:
Removed CachedObject from app manager views.

emord committed:
Removed caching invoice pdf.

emord committed:
Removed unused toggle.

dannyroberts committed:
remove app builder requisition mode

dannyroberts committed:
remove js model RequisitionConfigModel

dannyroberts committed:
remove docs, text, and comments relating to requisitions

orangejenny committed:
App builder v2: remove none and open-other case actions

orangejenny committed:
remove stray console statement

millerdev committed:
Turns out tags are better for various graph types

This allows top-list graphs for cases and fixtures.

NoahCarnahan committed:
Add another domain

dannyroberts committed:
remove unused imports

Commit:
do not allow ampersand

dannyroberts committed:
remove cloudcare filter_cases view

`cchq@hqproxy0:~/www/production/log$ grep -R cases/module .` gives nothing
since Mar 2017

orangejenny committed:
Merge branch 'master' into v2-limit-case-actions

dannyroberts committed:
remove cloudcare get_sessionand get_session_context views

`cchq@hqproxy0:~/www/production/log$ grep -R 'cloudcare.*api/sessions'` gives nothing

emord committed:
Added cached safe_download decorator.

dannyroberts committed:
remove getLocalizedString and localize unuzed cloudcare js utils

emord committed:
Added caching for some app manager endpoints.

emord committed:
Forced incorrect endpoints to fail hard.

emord committed:
Added caching to practice user endpoint.

emord committed:
Added hack for safe_download endpoints to allow caching.

dannyroberts committed:
rename touchforms-inline.* to formplayer-inline

proteusvacuum committed:
reset count when searching

proteusvacuum committed:
Add include closed to case search admin

snopoke committed:
stop processing UCR if column is missing

snopoke committed:
Merge pull request #17159 from dimagi/je/dont-use-cached-object

dont use cached object

snopoke committed:
Merge pull request #17168 from dimagi/dm/restore-stats

Turns out tags are better for various graph types

snopoke committed:
refactor tests to not use global app

snopoke committed:
make test actually work

snopoke committed:
remove invalid form xml since we're mocking the validation

emord committed:
Merge pull request #17176 from dimagi/sk/ucr-missing-column

stop processing UCR if column is missing

millerdev committed:
Track restore times for 'malawi-fp-study' domain

Commit:
ICDS - requested changes from July24/25

nickpell committed:
Merge pull request #17175 from dimagi/fr/search-admin-include-closed

Show closed cases in search admin

emord committed:
update staging.yaml

[ci skip]

emord committed:
update deploy module

emord committed:
Modify the response not the request.

emord committed:
Merge pull request #17180 from lwyszomi/icds_qa_feedback

ICDS - requested changes from July24/25

dannyroberts committed:
Merge branch 'master' into rename-touchforms-inline

dannyroberts committed:
rename inline touchform class and method

proteusvacuum committed:
Remove cached last claim

dannyroberts committed:
Merge pull request #17173 from dimagi/remove-unused-cloudcare-view

Remove unused cloudcare view

snopoke committed:
Create all build files when releasing app

dannyroberts committed:
remove unused imports

dannyroberts committed:
remove unused BaseSessionDataHelper.filter_cases

dannyroberts committed:
remove unused imports

dannyroberts committed:
remove unused get_open_form_sessions

millerdev committed:
Merge pull request #17181 from dimagi/dm/malawi-fp-study

Track restore times for 'malawi-fp-study' domain

dannyroberts committed:
remove unused cloudcare endpoint sync_db_api

found a single occurance of someone hitting it in prod proxy logs,
and that was on 2016/11/25

emord committed:
Check if the app is a build and is released.

orangejenny committed:
Fix case list breaking on language change

dannyroberts committed:
remove unused cloudcare endpoint render_form

biyeun committed:
Merge pull request #17167 from dimagi/nc/abt-fix

Abt report: Fix custom UCR expression for malformed submissions (again)

orangejenny committed:
Merge pull request #17166 from dimagi/v2-limit-case-actions

App builder v2: remove none and open-other case actions

orangejenny committed:
Don't show new app builder modal in v1

esoergel committed:
Don't translate built docs [ci skip]

dannyroberts committed:
Merge pull request #17165 from dimagi/remove-app-builder-requisition-mode

remove app builder requisition mode

dannyroberts committed:
Merge pull request #17172 from dimagi/rename-touchforms-inline

rename touchforms-inline.* to formplayer-inline

Commit:
Merge pull request #17134 from dimagi/wsp/fix-detail-continue

Hide detail tile continue correctly

esoergel committed:
Merge pull request #17190 from dimagi/es/translations

Don't translate built docs [ci skip]

emord committed:
Don't set session last visited domain for xml downloads.

emord committed:
Use anonymous user

proteusvacuum committed:
Merge pull request #17182 from dimagi/fr/cache-claim-key

Remove cached last claim

calellowitz committed:
Merge pull request #17185 from dimagi/fix-case-list

Fix case list breaking on language change

emord committed:
request.GET is immutable.

Commit:
typo

biyeun committed:
update submodule ref to prelogin

biyeun committed:
Update translations.

[ci skip]

orangejenny committed:
Merge pull request #17187 from dimagi/v1-drop-modal

Don't show new app builder modal in v1

dannyroberts committed:
Merge pull request #17188 from dimagi/prune-touchforms-api

Prune touchforms api

Commit:
use `'view_report'`

calellowitz committed:
Merge pull request #17164 from dimagi/unicode-fix-french-report-builder

fix report builder for unicode (french support)

emord committed:
Merge pull request #17171 from dimagi/je/cache-build-files

cache build files

emord committed:
update deploy module

proteusvacuum committed:
Merge pull request #17192 from dimagi/br/view-report-perm

Permissions.view_report

proteusvacuum committed:
Merge pull request #17169 from dimagi/br/do-not-allow-apersand

do not allow ampersand

orangejenny committed:
'Module' => 'Menu' in a few more places

orangejenny committed:
Wrong menu starred in advanced module form's case list dropdown

Commit:
Fix aggregation

Commit:
Revert "Fix aggregation"

This reverts commit 612c71220cb5fba6efb52d90a87b8569e0b0877f.

snopoke committed:
Merge pull request #17200 from dimagi/jls/stray-module-copy

'Module' => 'Menu' in a few more places

snopoke committed:
Merge pull request #17201 from dimagi/misdefined-module

Wrong menu starred in advanced module form's case list dropdown

snopoke committed:
Merge pull request #17148 from dimagi/sk/save-ledgers-django

save ledgers in django

snopoke committed:
Merge pull request #17183 from dimagi/sk/preemptively-create-build-files

Create all build files when releasing app

snopoke committed:
Merge pull request #17152 from dimagi/sk/riak-error

use correct exception

snopoke commented:
re-opening against master

snopoke commented:
replaced by https://github.com/dimagi/commcare-hq/pull/17207
